### PR TITLE
Add Ruby playground

### PR DESCRIPTION
**Pull request by TomKlotzPro**
This pull request add the ruby playground.

### CODE REVIEW

**Comment by TomKlotzPro** [comment]
I think we can .env and use it what do you think @nunomaduro

File: ruby/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Installation
+
+```bash
+git clone https://github.com/algolia/api-clients-playground
+```
+
+```bash
+cd api-clients-playground/ruby && bundle install
```
Comment (nunomaduro) [inline ruby/README.md]: What is bundle? If is a requirement you need to specify it. Concerning the template of the readme, can you update it to look exacly like this: https://github.com/algolia/api-clients-playground/pull/12/files#diff-41be41fc0c21938ff1ba8b2fdfa6291c.

File: ruby/Gemfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+# See https://github.com/algolia/algoliasearch-client-ruby 
+ gem "algoliasearch", ">=1.26"
+ gem 'dotenv'
```
Comment (nunomaduro) [inline ruby/Gemfile]: Why we need this?

File: ruby/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+Gemfile.lock
```
Comment (nunomaduro) [inline ruby/.gitignore]: Missing end of line.

File: ruby/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+```bash
+# Go inside the playground ruby folder
+cd api-clients-playground/ruby
+# Install dependencies
+bundle install
+```
+
+Finally, edit and run the code on `index.rb`:
+```
+ruby index.rb
```
Comment (nunomaduro) [inline ruby/README.md]: I think ruby is also a dependency. You need to specify it.

File: ruby/Gemfile (hunk 2 above)
Comment (TomKlotzPro) [reply ruby/Gemfile]: i was wondering if it was useful to use a .env file. But if we keep playground simple we should remove it

**Pull request merged**